docs/src/main/sphinx/installation/cli.rst

```diff
@@ -32,7 +32,10 @@ You can override your username with the ``--user`` option. It defaults to your
 operating system username. If your Trino server requires password
 authentication, use the ``--password`` option to have the CLI prompt for a
 password. You can set the ``TRINO_PASSWORD`` environment variable with the
-password value to avoid the prompt.
+password value to avoid the prompt. If you are using web browser sso
```

Comment:
Use new paragraph for this and then do 

```
The  ``--external-authentication`` option is used for browser-based SSO authentication, as detailed in :doc:`/security/oauth2`. With this configuration the CLI uses your web browser for authentication, and as a result this can only be used on client machines with a supported browser installed.
```


If you are using a browser-based SSO authentication flow,

Reply:
Ok so that's not 100% correct. The only requirement is that you need to have an access to a browser, so you can ssh on a remote machine and use cli from there but still be using the browser installed on your local pc. I don't want to give an impression that you can't be using this mode with ssh, which is probably one of the most frequent use case.

Reply:
Hm it seems to be a common scenario for an old style production maintenance. You have a ssh tunel opened up with very limited privileges to a production machine, and nothing else. Every interaction with services deployed there goes through this channel. That obviously does not scale up well, so it's not encouraged with heavily distributed systems, but the reality is that it's still a thing in many places.

Reply:
Maybe it is more common .. in either way though.. we have to describe how to actual use it